WebRoot/js/zimbraMail/prefs/view/ZmZimletsPage.js

```diff
@@ -555,11 +555,17 @@ ZmPrefZimletListView.prototype._getCellId = function(item, field, params) {
 	return DwtId.getListViewItemId(DwtId.WIDGET_ITEM_CELL, "zimlets", item.name, field);
 };
 
+ZmPrefZimletListView.prototype._getCellClass = function (item, field) {
+	if (field === ZmPrefZimletListView.COL_ACTIVE) {
+		return "ZmAlignCenter";
+	}
+};
+
 ZmPrefZimletListView.prototype._getHeaderList =
 function() {
 	var hlist = [
-		(new DwtListHeaderItem({field:ZmPrefZimletListView.COL_ACTIVE, text:ZmMsg.active, width:ZmMsg.COLUMN_WIDTH_ACTIVE})),
-        (new DwtListHeaderItem({field:ZmPrefZimletListView.COL_NAME, text:ZmMsg.name, width:ZmMsg.COLUMN_WIDTH_FOLDER_DLV})),    
+		(new DwtListHeaderItem({field:ZmPrefZimletListView.COL_ACTIVE, text:ZmMsg.active, width:ZmMsg.COLUMN_WIDTH_ZIMLET_ACTIVE})),
+        (new DwtListHeaderItem({field:ZmPrefZimletListView.COL_NAME, text:ZmMsg.name, width:ZmMsg.COLUMN_WIDTH_ZIMLET_NAME})),
```

Comment:
Yes, done. It was tabs vs spaces. Thanks !
https://github.com/Zimbra/zm-web-client/pull/98/commits/6a6a14dd2f05f961c3d5d8d962c046d40c19d080